fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -447,24 +447,20 @@ public void getData(
     /**
      * Builds the asynchronous workflows, and subscribes the appropriate channels to the appropriate workflows.
      *
-     * @param asyncAfter  How long the user is willing to wait for a synchronous request
-     * @param responseFormat  The requested format for the response
-     * @param uriInfo  The URI of the request
+     * @param apiRequest The apiRequest
```

Comment:
Param description doesn't help too much. People already know that from param name. Descriptions talks about what it is and how it's gonna be used